pkg/labels/filter.go

```diff
@@ -69,16 +69,10 @@ func DefaultLabelPrefixCfg() *LabelPrefixCfg {
 		Version: LPCfgFileVersion,
 		LabelPrefixes: []*LabelPrefix{
 			{
-				Prefix: common.GlobalLabelPrefix,
-				Source: common.ReservedLabelSource,
+				Prefix: "id.",
```

Comment:
`id.` is a short prefix that we can use in examples. `id.frontend` talks to `id.backend`